docs/VnVPlan/VnVPlan.tex

```diff
@@ -118,11 +123,11 @@ \subsection{SRS Verification Plan}
 
 \subsection{Design Verification Plan}
 
-\wss{Plans for design verification}
-
-\wss{The review will include reviews by your classmates}
-
-\wss{Create a checklists?}
+Design verification will be done by the developers of the project. The design will also be reviewed
+by the TAs of the course. The design of the system will be verified by going through the
+requirements from the SRS and determining whether the outputs correspond with the expected inputs.
+The verification will also be done by going through the MG and MIS checklist and ensure that all
+the modules are completed and fulfill the corresponding requirements.
```

Comment:
You should link to the SRS, MG, and MIS. Also use the full word with the acronym since this is the first time it is mentioned or put it in section 2.